### DISCUSSION TITLE
Using rectangle with automator

### DISCUSSION THREAD

**Discussion by arminbw** [rxhanson/Rectangle#701] [Ideas]
First of all thanks for creating Rectangle. It’s a great little app that I enjoy using every day.

I wonder if I could use it in a Mac OS Automator Application. I would like to launch several apps in one fell swoop and I would like to automatically position them using Rectangle (mail gets 2 thirds left then hides, terminal gets 2 halfs right etc.). Has anyone done this before?

**Comment by rxhanson** [comment]
This would require making the app scriptable. I have looked into this a couple times but it's a surprisingly complex task that I have not had the bandwidth to add in. Pull requests for it are welcome, though!

**Comment by arminbw** [comment]
If it’s overly complex, it’s definitely not worth the effort atm!
I will try to add some basic windows positioning to the automator script itself.
